src/custom/utils/operator/index.ts

```diff
@@ -251,5 +252,18 @@ export async function getOrder(chainId: ChainId, orderId: string): Promise<Order
   }
 }
 
+export async function getBestPriceQuote(params: PriceQuoteParams) {
```

Comment:
This is confusing.

The function is used on `_getBestPriceQuote`.

But both are doing kind of different things.

This gets all prices from multiple endpoints while the other gets a single price from all sources.

Maybe this should be called `getAllPriceQuotes`

Reply:
Addressed in #882